src/MySqlConnector/Serialization/MySqlSession.cs

```diff
@@ -526,47 +526,81 @@ private void VerifyConnected()
 
 		private async Task InitSslAsync(ProtocolCapabilities serverCapabilities, ConnectionSettings cs, IOBehavior ioBehavior, CancellationToken cancellationToken)
 		{
-			X509CertificateCollection clientCertificates = null;
+			X509CertificateCollection CertificateFiles = null;
```

Comment:
This still only contains (a single) client certificate, correct? If so, I think the old name of `clientCertificates` is clearer.